java/arcs/core/crdt/CrdtSet.kt

```diff
@@ -21,17 +21,27 @@ import arcs.core.crdt.internal.Actor
 import arcs.core.crdt.internal.VersionMap
 
 /** A [CrdtModel] capable of managing a set of items [T]. */
-class CrdtSet<T : Referencable>(
+class CrdtSet<T : Referencable> (
```

Comment:
Nit: the space introduced here isn't needed.

Reply:
removed